NEWS.md

```diff
@@ -1,3 +1,6 @@
+## 2.0.0 2021-03-10
```

Comment:
Just a note, we'll need to correct this date when we do the actual release.

Reply:
OK